assets/js/components/ControlPanel.tsx

```diff
@@ -147,13 +192,36 @@ const ControlPanel = ({ clientName, webrtc, playerCallback }: ControlPanelProps)
 
   return (
     <>
-      <Box borderWidth="1px" width="100%" min-height="40px" padding="10px">
-        {getDropdownButton("audio")}
-        {getDropdownButton("video")}
-        {getMuteButton("audio")}
-        {getMuteButton("video")}
-      </Box>
-      <button onClick={() => shareScreen(webrtc, clientName, playerCallback)}>shareScreen</button>
+      <div className="ControlPanel">
+        <GenericButton icon={<Settings className="PanelButton" />} onClick={onOpen} />
+        <div className="CenterIcons">
+          {getMuteButton("video", Cam, CamDisabled)}
+          {getMuteButton("audio", Microphone, MicrophoneDisabled)}
+          <GenericButton
+            icon={
+              <PhoneDown
+                className="DisconnectButton"
+                color={style.getPropertyValue("--bg-light-color-1")}
```

Comment:
I would move it also to file which will return fontColor.

Reply:
Fixed.